benchalerts/parse_conbench.py

```diff
@@ -113,11 +105,13 @@ def regression_details(all_comparisons: dict) -> str:
     return details
 
 
-def regression_check_status(all_comparisons: dict) -> GitHubRepoClient.CheckStatus:
+def regression_check_status(
+    comparisons: List[ConbenchClient.RunComparison],
+) -> GitHubRepoClient.CheckStatus:
```

Comment:
Ah, we've already got nested classes. Hm. I think I'd rather have them namespaced by file (i.e. module) rather than by class; this seems like a workaround to create namespaces

Reply:
Yeah, with the change I mentioned above, I can also move `CheckStatus` out of `GitHubRepoClient` in the same file. I think that will be fine.